mono/metadata/object-internals.h

```diff
@@ -2019,4 +2042,9 @@ mono_class_get_virtual_method (MonoClass *klass, MonoMethod *method, gboolean is
 MonoStringHandle
 mono_string_empty_handle (MonoDomain *domain);
 
+// FIXME value can be a raw pointer
+// We probably need mono_field_set_value_handle_ref and mono_field_set_value_handle_value
+void
+mono_field_set_value_handle (MonoObjectHandle obj, MonoClassField *field, void *value);
```

Comment:
If this signature is going to be wrong (because `void *value` could be a raw ptr to managed), let's not add it - just put inline its body at the use site in `custom-attr.c` and leave a FIXME.

Reply:
ok. I'm not sure my comment is correct, but ok either way.

Reply:
ps: do we need fixme anyway? I think there are like two phases here: The large obvious phase we are now in, and the later also maybe large mop-up driven by Centrinel. I think Centrinel will superset any few fixmes I leave.